samples/graph-appcatalog-lifecycle/csharp/AppCatalogSample/Helper/OAuthHelpers.cs

```diff
@@ -0,0 +1,262 @@
+﻿using AdaptiveCards;
+using Microsoft.Bot.Builder;
+using Microsoft.Bot.Schema;
+using Microsoft.Graph;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace AppCatalogSample.Helper
+{
+    public static class OAuthHelpers
```

Comment:
Not sure why this is called as OAuthHelper, could you please correct the naming for this class?